api-report/container-runtime.api.md

```diff
@@ -448,6 +448,14 @@ export interface IProvideSummarizer {
     readonly ISummarizer: ISummarizer;
 }
 
+// @public
+export interface IRefreshSummaryAckOptions {
+    readonly ackHandle: string;
+    readonly proposalHandle: string | undefined;
+    readonly summaryLogger: ITelemetryLogger;
```

Comment:
I don't think logger should be part of this.

Reply:
yes, I considered that but the other interfaces do add the logger....